src/simulate/assets/articulation_body.py

```diff
@@ -139,3 +139,35 @@ def __post_init__(self):
         if self.inertia_tensor is not None:
             if len(self.inertia_tensor) != 3:
                 raise ValueError("inertia_tensor must be a list of 3 floats")
+
+    def copy(self) -> "ArticulationBodyComponent":
+        """
+        Create a copy of this rigid body with the same values.
+
+        Returns:
+            copy (`RigidBodyComponent`):
+                A copy of this rigid body
+        """
+        instance_copy = ArticulationBodyComponent(
```

Comment:
We may make these class evolve in the future and we'll miss it here probably.

Let's either use a loop over the fields of the dataclass and/or add a test doing such loop and checking that we didn't forget anything?

Reply:
Fixed